apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -845,6 +995,290 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
             "console:manage.features.user.modals.bulkImportUserWizard.wizardSummary.tableStatus.failed" );
     };
 
+    /**
+     * Render Multiple Users mode selection section.
+     */
+    const resolveMultipleUsersModeSelection = (): ReactElement => {
+        return(
+            <Grid.Row>
+                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                    <Button.Group
+                        size="large"
+                        labeled
+                        basic
+                    >
+                        {
+                            Object.values(MultipleInvites).map((mode: string, index: number) => {
+                                return(
+                                    <>
+                                        <Popup
+                                            trigger={ (
+                                                <div className={ "inline-button" } >
+                                                    <Button
+                                                        disabled={ 
+                                                            isAlphanumericUsername
+                                                            && mode === MultipleInvites.MANUAL
+                                                        }
+                                                        key={ index }
+                                                        active={ configureMode === mode }
+                                                        className="multiple-users-config-mode-wizard-tab"
+                                                        content={
+                                                            UserManagementUtils.resolveMultipleInvitesDisplayName(
+                                                            mode as MultipleInvites
+                                                            )
+                                                        }
+                                                        onClick={ (
+                                                            event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+                                                            event.preventDefault();
```

Comment:
In this new function, is it possible to use the oxygen-ui components? Without relying on semantic ui components ?